sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### Other Changes
 
+- Added several `ostream` insertion operators for AMQP types.
```

Comment:
Same question, the types I've seen these operators being added for are all in `_detail` namespace, if I am not mistaking.
Do you want to put it as more generic "expanded logging coverage"?

Reply:
Those types shouldn't really be in the public headers, but they *ARE* in the public headers for now, even though they're in the _detail namespace.

As such, they're developer facing and should be called out in the log.

Reply:
Types in the `_detail` namespace are **not** developer facing (i.e. not intended to be used by customers), as per our docs and guidelines, which is why they aren't even included as part of our doxygen docs (even if they are technically publicly visible).

Mentioning changes to the API surface for such types in the changelog is introducing noise that isn't relevant to that audience. It sends mixed messages for us to document and highlight "implementation detail" changes in the changelogs to the customer.

Reply:
I disagree, but don't feel strongly about it. I'll happily approve a pull request which removes that line from core.